WebCryptoAPI/generateKey/failures.js

```diff
@@ -98,7 +94,7 @@ function run_test(algorithmNames) {
                 results.push({name: algorithmName, hash: "SHA-256", modulusLength: 1024, publicExponent: publicExponent});
             });
         } else if (algorithmName.toUpperCase().substring(0, 2) === "EC") {
-            ["P-512", "Curve25519"].forEach(function(curveName) {
+            ["P-512"].forEach(function(curveName) {
```

Comment:
Ditto here, it might be good to explain in the commit message why we should do this.
AIUI, now that we support "Curve25519", it should no longer be a bad algorithm.